src/platform/backends/qemu/dnsmasq_process_spec.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2019 Canonical, Ltd.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; version 3.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef MULTIPASS_DNSMASQ_PROCESS_SPEC_H
+#define MULTIPASS_DNSMASQ_PROCESS_SPEC_H
+
+#include <multipass/ip_address.h>
+#include <multipass/optional.h>
+#include <multipass/path.h>
+#include "process_spec.h"
+
+#include <QDir>
```

Comment:
We have `mp::Path`, think we should use that instead?

Reply:
Done